### PR TITLE
PLAT-699 Update tab title when changing pages

### PR DESCRIPTION
**Pull request by daniel-klose**
image

### CODE REVIEW

File: platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java [hunk 1]
```diff
@@ -489,6 +489,12 @@ public AjaxResourceLinkRegistry getResourceLinkRegistry() {
 
 	// -------------------------------- browser manipulation -------------------------------- //
 
+	@Override
+	public void updateDocumentTitle(String pageTitle) {
```
Comment (oliver-richers) [inline platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java:493]: why not `setDocumentTitle(IDisplayString)`? wouldn't that be better

File: platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java [hunk 2]
```diff
@@ -536,6 +542,11 @@ private void JS_removeNodeAttribute(IDomNode node, String name) {
 		updateCodeJS.appendStatement(JS_getNode(node) + ".removeAttribute('" + name + "');");
 	}
 
+	private void JS_setAttribute(String attribute, String value) {
```
Comment (oliver-richers) [inline platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java:545]: hmm, isn't this method redundant? shouldn't be the first occasion where we set an attribute

**Review by oliver-richers — CHANGES_REQUESTED** [review]

File: platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java (hunk 1 above)
Comment (daniel-klose) [reply platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java:493]: dunno, changed to set

File: platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java (hunk 2 above)
Comment (daniel-klose) [reply platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java:545]: I did not find any such method, please give an example :)

Comment (oliver-richers) [reply platform-ajax/src/main/java/com/softicar/platform/ajax/engine/AjaxDomEngine.java:545]: ok

**Review by oliver-richers — APPROVED** [review]

**Pull request merged**